api-client/src/stringifyFormData.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+type FormDataType = FormData
+  | {| getParts: () => Array<{ string: string, fieldName: string }> |}
+
+export function stringifyFormData (formData: FormDataType): string {
```

Comment:
Maybe a cmment would be nice here: `stringifies FormData from react-native or the browser variant of it`